torch_xla/experimental/plugins.py

```diff
@@ -65,3 +74,18 @@ def default() -> DevicePlugin:
 def register_plugin(name: str, device_plugin: DevicePlugin):
   _plugin_registry[name.upper()] = device_plugin
   torch_xla._XLAC._register_pjrt_plugin(name, device_plugin.library_path())
+
+
+def register_installed_plugins():
+  pjrt_entry_points = importlib_metadata.entry_points(group='torch_xla.plugins')
+  for ep in pjrt_entry_points:
+    device_plugin_class = ep.load()
+
+    # HACK: TpuPlugin raises EnvironmentError if libtpu is not installed.
+    # TODO(wcromar): Device if catching `EnvironmentError` is a permanent
```

Comment:
nit: you meant "decide" if catching `EnvironmentError` is a permanent

Reply:
:facepalm:  fixed